unattended_installer/install_functions/checks.sh

```diff
@@ -346,7 +358,7 @@ function checks_ports() {
 
     if [ "${used_port}" -eq 1 ]; then
         common_logger "The installation can not continue due to port usage by other processes."
-        installCommon_rollBack
+        installCommon_rollBack 
```

Comment:
Remove the trailing character, please.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2469/commits/d5e7df8fb9bac4b4a6ec8168f1f2e29910dc7354.